scripts/devtools/tpl-manager.py

```diff
@@ -149,12 +170,19 @@ def build_deps(args):
   # Loop through the specs we want TPLs for and build/install/get them as necessary.
   if not args.init_only:
       for s in spec_list:
+
         print("** Building TPL's and generating host-config for {0}%{1} ...".format(package_name,s))
         os.environ["SPEC"] = s
         os.environ["LC_ALL"] = "en_US.UTF-8"
+
         if not args.no_spec:
             if sexe("{0} spec --fresh -I {1}@develop%{2}".format(spack_cmd, package_name, s), echo=True) : sys.exit(1)
-        if sexe("{0} dev-build --fresh --quiet --deprecated -u initconfig {2}@develop%{3} 2>&1 | tee -a \"dev-build-{3}-out.txt\"".format(spack_cmd, os.getcwd(), package_name, s), echo=True) : sys.exit(1)
+
+        # Install only the dependencies for Spheral
+        if sexe("{0} install --fail-fast --fresh --deprecated --only dependencies {2}@develop%{3} 2>&1 | tee -a \"dev-build-{3}-out.txt\"".format(spack_cmd, os.getcwd(), package_name, s), echo=True) : sys.exit(1)
```

Comment:
Why wasn't this install call needed before?

Reply:
`dev-build` acts the same as `install` in regards to concretizing and installing dependencies, but `dev-build` doesn't have the `--fail-fast` option so we can have spack install only the dependencies with `install` and then let it create the `host-config` file from `dev-build` with our local repo. Using `install` tries to pull spheral develop and that isn't what we want for tpl-manager